core/src/sql/v1/value/value.rs

```diff
@@ -2848,16 +2848,20 @@ mod tests {
 
 	#[test]
 	fn check_size() {
-		assert!(64 <= std::mem::size_of::<Value>());
-		assert_eq!(104, std::mem::size_of::<Error>());
-		assert_eq!(104, std::mem::size_of::<Result<Value, Error>>());
+		assert!(
+			64 >= std::mem::size_of::<Value>(),
+			"expected Value to be smaller then 64 bytes found {:?}",
+			std::mem::size_of::<Value>()
+		);
+		assert_eq!(112, std::mem::size_of::<Error>());
+		assert_eq!(112, std::mem::size_of::<Result<Value, Error>>());
```

Comment:
This test failed, some sizes seemed to have changed even though this PR makes no changes to any of these structs.